### DISCUSSION TITLE
deploying Tailwind with Next.js to Vercel not worked

### DISCUSSION THREAD

**Discussion by rikusen0335** [vercel/next.js#20960] [Help] [answered]
Repo here: https://github.com/rikusen0335/miso-host

I tried few way and deployed this but Tailwind still not working. Did I something wrong with this?

Edit: Progressions: I confirmed that production build isn't work too

**Comment by rikusen0335** [comment, marked as answer]
This was wrong with `tailwind.config.js`.
My `purge` path was: `purge: ['./components/**/*.{js,ts,jsx,tsx}', './pages/**/*.{js,ts,jsx,tsx}']`,
but correct one is `purge: ['./src/components/**/*.{js,ts,jsx,tsx}', './src/pages/**/*.{js,ts,jsx,tsx}']`

**Reply by jorgearuv** [reply]
This fixed my issue, thank you!

**Reply by rikusen0335** [reply]
I appriciate this helps your issue :DD
I lost about 3 hours for this stupid mistake ;(((

**Reply by Arvelon** [reply]
Thanks!
